unittest/quaternion.cpp

```diff
@@ -28,5 +28,19 @@ BOOST_AUTO_TEST_CASE(test_assignQuaternion)
   }
 }
 
+BOOST_AUTO_TEST_CASE(test_uniformRandom)
+{
+  srand(0);
+
+  using namespace pinocchio;
+  Eigen::Quaternion<double> q;
+
+  for (int i = 0; i < 1 << 10; ++i) {
```

Comment:
What is `i < 1 << 10`?

Reply:
I find it more readable than `1 < 10000000000`.

Reply:
I will change it to `i < (1 << 10)`. I don't know what operator between `<` and `<<` has precedence.

Reply:
OK. Thanks for the reply.

Reply:
Next time, you can simply use 1e10.